roles/container-engine/docker/tasks/pre-upgrade.yml

```diff
@@ -15,6 +15,17 @@
     - ansible_os_family == 'Debian'
     - (docker_versioned_pkg[docker_version | string] is search('docker-ce'))
 
+
+- name: Ensure podman not installed. | RedHat
+  package:
+    name: '{{ podman_remove_packages_yum }}'
+    state: absent
+  when:
+    - ansible_os_family == 'RedHat'
+    - (docker_versioned_pkg[docker_version | string] is search('docker-ce'))
```

Comment:
The pull request message says both containerd.io and docker-ce can be conflict against podman.
Why doesn't here check containerd?
Isn't it enough to just check like
```
- container_manager in ['docker', 'containerd']
```
?

Reply:
`containerd.io` is  part/dependency of `docker-ce` defined in `roles/container-engine/docker/vars/*.yml`
So condition check by `docker-ce` will be enough.
 moreover, this `pre-upgrade.yml`  belongs to  `roles/container-engine/docker` , this folder is for `docker`.
When user wants to install `containerd` only without docker, whether it will conflict with podman ?  I've no chance to test it so far, and this problem belongs to `roles/container-engine/containerd` sub folder. 

And the original `pre-upgrade.yml` separate Debian & RedHat into 2  `Ensure old versions of Docker are not installed` task. I think only one `package with state absent` is enough.  But I don't want to involve multiple change into one PR and left it as it is.

Reply:
Thank you for your explanation, I see your point.

/ok-to-test